OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -120,12 +120,14 @@ public override bool HandleMouseInput(MouseInput mi)
 							}
 						}
 					}
-					else if (dragStart.HasValue)
+					else if (dragStart.HasValue && (!(World.OrderGenerator is GenericSelectTarget) || hasBox))
```

Comment:
Ok, I figured it would be something like that.  Could you please add a comment explaining this?